README.md

```diff
@@ -1,3 +1,52 @@
+
+All of our major initiatives are driven by volunteers. Often those are collaborating across organisational and corporate boundaries. That way together we can grow beyond what any single organisation would be able to reach.
+
+# Getting Involved
+
+So you are interested getting involved with the InnerSource Commons. Essentially there are two ways for you to support our cause:
+
+## Donate time
+
+Your first step is to get in touch with the InnerSource Community. You will find most of us hanging out on the [InnerSource Commons Slack](https://innersourcecommons.slack.com/join/shared_invite/zt-277nb21f5-nnwqJU7gPWTG7KE~uGFt_w#/shared-invite/email). Look for the #general channel to introduce yourself, share your InnerSource story. We appreciate learning more on what you're doing with InnerSource. You can also join our #virtual-coffee-buddies to get you in touch 1-1 with one other community participant chosen at random from that channel. If you want to get notified of mentions of InnerSource on the web feel free to join #track_isc_google_alerts. If you have jobs to share with the community or are looking for a job there's a separate #jobs channel.
+
+In addition look for channels related to our working groups. As of early 2024 those are the Learning Path, Patterns, and ISPO group. Channels are named accordingly: #learning-path, #innersource-patterns, #ispo-working-group. In addition each of the groups has their own GitHub repository: [Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath), [Patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns), [ISPO](https://github.com/InnerSourceCommons/ispo-working-group).
+
+If you want to help with outreach and marketing, feel free to join on Slack the #marketing channel.
+
+As a general rule of thumb: There's no expectation at all that new contributors start by sending us patches. Much like in open source you are invited to join our communication channels and spend some time watching over the shoulder. After a little while you will find yourself starting to answer incoming questions, provide insights to discussions initiated. Those are already very valuable contributions! Over time you may find yourself making modifications to our existing material - this can be as small as adding your organisation to the known instances section of a pattern, or as large as creating a learning path segment that you need for rolling out InnerSource in your organisation. Chances are high that you will find others in need for that material as well - and willing to collaborate on the material together with you.
+
+Donating time is the only way for you to become a member of the foundation and gain voting rights on new members and board members. One caveat though: All of the work shaping InnerSource happens withing the working groups. The foundation exists to enable those groups and provide some shelter from whatever legal risk there may be from working in the public.
```

Comment:
Typo here ... "withing"

Reply:
@michael-basil, GitHub review tip, you can propose changes like this:
```suggestion
Donating time is the only way for you to become a member of the foundation and gain voting rights on new members and board members. One caveat though: All of the work shaping InnerSource happens within the working groups. The foundation exists to enable those groups and provide some shelter from whatever legal risk there may be from working in the public.
```
This is how:
<img width="816" alt="image" src="https://github.com/InnerSourceCommons/foundation-governance/assets/54676965/f7f5b891-c7dc-484f-93e6-509b0f845642">

It makes it easy for the PR author to integrate that with a commit directly from the review UI.